sdk/communication/Azure.Communication.PhoneNumbers/tests/PhoneNumbersClient/PhoneNumbersClientLiveTests.cs

```diff
@@ -528,6 +528,72 @@ public void GetPurchasedPhoneNumbersNextPage()
             Assert.NotNull(purchasedPhoneNumbers);
         }
 
+        [Test]
+        [AsyncOnly]
+        public async Task GetPurchasedPhoneNumbersAsPagesAsync()
+        {
+            var client = CreateClient();
+            var phoneNumbers = await client.GetPurchasedPhoneNumbersAsync().ToEnumerableAsync();
+            var phoneNumbersCount = phoneNumbers.Count;
+
+            if (phoneNumbersCount < 2)
```

Comment:
If phone number counts is 0 or 1, we should still want to be able to test this. Apply this to the other tests as well.

If 0 phone numbers, page size is 0.
If 1 phone number, page size is 1.